### ISSUE TITLE
Bump Color IR Layer to 2024

### ISSUE THREAD

**Issue by stdavis** [agrc/kitchen-sink#285]
@rkelson Reminded us that this needs to be bumped to the 2024 layer in a few months.

**Comment by stdavis** [comment]
/remind me to check on this in 3 months

**Comment by steveoh** [comment]
/remind @stdavis to check on this in 3 months

**Comment by rkelson** [comment]
yeah let's check back in a few months but I just remembered the naip service will likely be last in line behind NAIP county mr. sid downloads and hexagon service updates considering the red tape with the final naip product from the feds.

**Comment by stdavis** [comment]
@rkelson Hows this looking? Are we ready to update yet?

**Comment by rkelson** [comment]
Sanborn held off on the high-res HxGN update considering folks were out of the office for the holiday. I was told they plan to roll out the 2024 imagery starting today 12/2/24.

there is no Color IR layer associated with the HxGN high-res RGB services. Maybe we should have a talk. I am going to check now on the status of the '24 NAIP which has a color IR layer.

**Comment by stdavis** [comment]
This should be done after we create the NAIP caches on honeycomb.

**Comment by rkelson** [comment]
Nope and I have checked twice that I am on the list to be notified when the '24 NAIP is available

**Comment by stdavis** [comment]
@rkelson Just checking in on this again. Is there any information on a timeframe?

**Comment by rkelson** [comment]
@stdavis I talked to the fellow that sends me the NAIP hard drives a couple months ago, which was the 3rd follow up, and he said they 'pulled the plug' on the way they used to create the .tif tiles and maybe something else would replace it. I have a feeling that part of the NAIP program got DOGEed.
